endPoint/httpHostEndPoint.js

```diff
@@ -295,6 +311,13 @@ class HttpHostEndPoint extends HostEndPoint {
       );
     }
   }
+  sanitizeInput(input) {
+    let inputStr = String(input);
+    inputStr = inputStr.replace(/[\r\n]/g, '');
```

Comment:
What about `%0d` and `%0a`?
Also blank id not good choice.I think a space is better.